deployment/stacks/parent.ts

```diff
@@ -49,49 +43,56 @@ export class ParentStack extends Stack {
       customResourceRole,
     });
 
-    const { apiGatewayRestApiUrl, restApiId } = createApiComponents({
-      ...commonProps,
-      tables
-    });
-
-    if (!isLocalStack) {
-      const {
-        applicationEndpointUrl,
-        distribution,
-        uiBucket,
-      } = createUiComponents({ ...commonProps });
-
-      const {
-        userPoolDomainName,
-        identityPoolId,
-        userPoolId,
-        userPoolClientId,
-      } = createUiAuthComponents({
+    if (isLocalStack) {
+      createApiComponents({
         ...commonProps,
-        applicationEndpointUrl,
-        restApiId,
-        customResourceRole
+        tables,
       });
+      return;
```

Comment:
I hope you'll indulge me with a long-winded comment, explaining why we bail out with an early return here:
```suggestion
      /*
       * For local dev, the LocalStack container will host the database and API.
       * The UI will self-host, so we don't need to tell CDK anything about it.
       * Also, we skip authorization locally. So we don't set up Cognito,
       * or configure the API to interact with it. Therefore, we're done.
       */
      return;
```